KarhooUISDK/UIComponents/LegalNotice/LegalNoticeViewController.swift

```diff
@@ -18,7 +19,7 @@ public struct KHLegalNoticeViewID {
 final class LegalNoticeViewController: UIViewController, BaseViewController {
     
     private var zeroHeightTextConstreint: NSLayoutConstraint!
-    private let shouldShowView = UITexts.Booking.legalNoticeText.isNotEmpty
+    private let shouldShowView = true // UITexts.Booking.legalNoticeText.isNotEmpty
```

Comment:
This will need to be reverted before merging into the feature branch. I'd be ok with it existing until we merge into develop if we didn't run the risk to forget about it. Better make this change again then just for the duration of testing, than to have this get into production by mistake.